Makefile

```diff
@@ -1,14 +1,36 @@
+# https://www.gnu.org/software/make/manual/html_node/Special-Variables.html#Special-Variables
+.DEFAULT_GOAL := help
+
 #----------------------------------------------------------------------------------
-# Base
+# Help
 #----------------------------------------------------------------------------------
+# Our Makefile is quite large, and hard to reason through
```

Comment:
nit: as we go through adding already large prs lets try to break out smaller concerns like this separately. Given review fatigue we might not check these portions as they should and after the squash there wont be the same history as we might want for things like this.

Reply:
Yeah that's fair, I figured since I was adding targets here it made sense to change them all at once, but I'll try to do that moving forward